app/views/admin/products_v3/_variant_row.html.haml

```diff
@@ -48,11 +48,15 @@
 %td.align-left
   -# empty
 %td.align-right
-  - if variant.persisted?
-    = render(VerticalEllipsisMenu::Component.new) do
+  = render(VerticalEllipsisMenu::Component.new) do
+    - if variant.persisted?
       = link_to t('admin.products_page.actions.edit'), edit_admin_product_variant_path(variant.product, variant)
       - if variant.product.variants.size > 1
         %a{ "data-controller": "modal-link", "data-action": "click->modal-link#setModalDataSetOnConfirm click->modal-link#open",
             "data-modal-link-target-value": "variant-delete-modal", "class": "delete",
             "data-modal-link-modal-dataset-value": {'data-current-id': variant.id}.to_json }
           = t('admin.products_page.actions.delete')
+    - else
+      %a{ 'data-action': "nested-form#remove" }
+        = f.hidden_field :_destroy
```

Comment:
I hadn't seen a field inside an `<a>` before. I guess it works if specs pass!

Reply:
Aah, it works but it is not valid html. I have updated the haml and put the hidden field out of the anchor.